orderable/tests/test_models.py

```diff
@@ -207,3 +209,10 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
+    def test_save_subtask(self, sort_orders):
+        task = Task.objects.create()
+
+        for order in sort_orders:
+            SubTask.objects.create(task=task, sort_order=order)
```

Comment:
This doesn't occur with the `Task` model because it doesn't have any `unique` constraints.